test/components/notification.service.test.js

```diff
@@ -38,10 +38,18 @@ describe('Factory: Nofification', function() {
 
     it("allows message context to be specified for interpolation", function () {
         var message = "Everything ran correctly {{ ending }}!",
+            options = {link: ""},
             $scope = {ending: 'yay'},
```

Comment:
Its not obvious, but you'll have to put the context in the options
`options = {link: "", context: {ending: yay}}`
then call as `Notification.setSuccessMessage(message, options)`